Gopkg.toml

```diff
@@ -30,12 +30,8 @@
   revision = "fe8f9c4593d05f029afeb2c08c3e220120306506"
 
 [[constraint]]
-  name = "github.com/sensu/sensu-go"
```

Comment:
https://github.com/sensu/sensu-slack-handler/pull/19#discussion_r289935497

Reply:
Correct me if I am wrong. I think github.com/sensu/sensu-plugins-go-library library will pull in github.com/sensu/sensu-go as it has dependency on it.

Reply:
Yes, sensu-plugins-go-library will pull in sensu-go as a dependency. However this repository, sensu-pagerduty-handler is _directly_ dependent on the sensu-go api (specifically it imports corev2.Event). If there were a change to that api in sensu-go that is required for this handler, the dependency would have to be bumped in sensu-plugins-go-library first.

While this does compile, sensu-go should be treated as a direct dependency, rather than a transitive one (see https://github.com/golang/dep/blob/master/docs/FAQ.md#what-is-a-direct-or-transitive-dependency).

Reply:
Makes sense. Added the direct dependency in the latest commit.